eng/packaging.targets

```diff
@@ -1,309 +1,294 @@
-    </ItemGroup>
-  </Target>
-
-  <!-- Call a target in the analyzer project to get all the files it would normally place in a package.
-       These will be returned as items with identity pointing to the built file, and PackagePath metadata
-       set to their location in the package. IsSymbol metadata will be set to distinguish symbols. -->
-  <Target Name="IncludeAnalyzersInPackage"
-          Condition="'@(ProjectReference)' != '' and @(ProjectReference->AnyHaveMetadataValue('PackAsAnalyzer', 'true'))">
-    <MSBuild Projects="@(ProjectReference->WithMetadataValue('PackAsAnalyzer', 'true'))"
-             Targets="GetAnalyzerPackFiles"
-             RemoveProperties="SetTargetFramework">
-      <Output TaskParameter="TargetOutputs" ItemName="_AnalyzerFile" />
-    </MSBuild>
-
-    <ItemGroup>
-      <Content Include="@(_AnalyzerFile)" Pack="True" Condition="!%(_AnalyzerFile.IsSymbol)" />
-      <!-- Symbols don't honor PackagePath.  By default they are placed in lib/%(TargetFramework).
-           Pack does honor TargetPath and does Path.Combine("lib/%(TargetFramework)", "%(TargetPath)"),
-           so a rooted path value for TargetPath will override lib.
-           https://github.com/NuGet/Home/issues/10860 -->
-      <_TargetPathsToSymbols Include="@(_AnalyzerFile)" TargetPath="/%(_AnalyzerFile.PackagePath)" Condition="%(_AnalyzerFile.IsSymbol)" />
-    </ItemGroup>
-  </Target>
-
-  <PropertyGroup>
-    <_MultiTargetRoslynComponentTargetsTemplate>$(MSBuildThisFileDirectory)MultiTargetRoslynComponent.targets.template</_MultiTargetRoslynComponentTargetsTemplate>
-    <MultiTargetRoslynComponentTargetsFileIntermediatePath>$(IntermediateOutputPath)MultiTargetRoslynComponent.targets</MultiTargetRoslynComponentTargetsFileIntermediatePath>
-    <IncludeMultiTargetRoslynComponentTargets Condition="'$(IncludeMultiTargetRoslynComponentTargets)' == ''">true</IncludeMultiTargetRoslynComponentTargets>
-  </PropertyGroup>
-
-  <!-- In packages that contain multi-target Analyzers, include a .targets file that will select the correct analyzer. -->
-  <Target Name="IncludeMultiTargetRoslynComponentTargetsInPackage"
-          AfterTargets="IncludeAnalyzersInPackage"
-          Condition="'@(ProjectReference)' != '' and
-                     @(ProjectReference->AnyHaveMetadataValue('PackAsAnalyzer', 'true')) and
-                     '$(IncludeMultiTargetRoslynComponentTargets)' == 'true'"
-          DependsOnTargets="GenerateMultiTargetRoslynComponentTargetsFile">
-    <ItemGroup>
-      <Content Include="$(MultiTargetRoslynComponentTargetsFileIntermediatePath)" PackagePath="buildTransitive\netstandard2.0\$(PackageId).targets" />
-      <Content Include="$(MultiTargetRoslynComponentTargetsFileIntermediatePath)" PackagePath="buildTransitive\%(NETStandardCompatError.Supported)\$(PackageId).targets" Condition="'@(NETStandardCompatError)' != ''" />
-    </ItemGroup>
-  </Target>
-
-  <Target Name="WarnOnProjectReferenceToFrameworkAssemblies"
```

Comment:
Can you double check that NuGet is getting the right data for pruning when we use our live-built framework?

Reply:
Good point. I just added a commit that generates the PackageOverrides.txt file as part of the `libs.sfx` subset.